### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by mgkram4**
Hello, the homepage has three sections: header, middle, and end—also a base navbar and footer.

### CODE REVIEW

File: public/images/googeldevgroup.png
Comment (blbergo) [inline public/images/googeldevgroup.png:1]: rename it to be `google-dev-group.png` for readability

File: public/images/googleIcon1.png
Comment (blbergo) [inline public/images/googleIcon1.png:1]: same for the following

File: src/app/homepage/page.tsx
Comment (blbergo) [inline src/app/homepage/page.tsx:1]: this file belongs under just /app - `page.tsx` will route to localhost:3000/

File: src/app/homepage/sections/End.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import Image from "next/image";
+
+export default function End() {
+  return (
+    <div className="flex flex-col items-center justify-center bg-dark-400 text-center p-4">
+      {/* SECTION CHIP*/}
+      <div
+        className="
+        mt-20
```
Comment (blbergo) [inline src/app/homepage/sections/End.tsx:9]: formatting here

File: src/app/homepage/sections/End.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          className="
+          rounded-full
+          bg-dark-400
+          px-4
+          py-1
+        "
+        >
+          <p
+            className="
+
```
Comment (blbergo) [inline src/app/homepage/sections/End.tsx:27]: extra whitespace (we get charged based on file size on FB )

File: src/app/homepage/sections/End.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            bg-gradient-to-r from-[#fbbc04] via-[#f46930] to-[#eb4235]
+            text-transparent bg-clip-text
+          "
+          >
+            About
+          </p>
+        </div>
+      </div>
+
+      {/* HEADER + DESCRIPTION */}
```
Comment (blbergo) [inline src/app/homepage/sections/End.tsx:38]: great descriptions btw

File: src/app/homepage/sections/End.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+            <p className="text-body-lg md:text-body-xl ">
+              Our developer community spans across the globe. Connect through
+              chapter-hosted events, join a program to fast-track success, or
+              simply explore this space to learn something new from the
+              community.
+            </p>
+          </div>
+        </div>
+        <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4">
+          {["OC", "SD", "LA"].map((city) => (
```
Comment (blbergo) [inline src/app/homepage/sections/End.tsx:69]: can pass these in as a prop instead, but great job simplifying repeated work

File: src/app/homepage/sections/End.tsx
Comment (blbergo) [inline src/app/homepage/sections/End.tsx:1]: move these to `src/app/components/home` - current setup will create routes for each component here

File: src/app/homepage/sections/Header.tsx
Comment (blbergo) [inline src/app/homepage/sections/Header.tsx:1]: same here

File: src/app/homepage/sections/Middle.tsx
Comment (blbergo) [inline src/app/homepage/sections/Middle.tsx:1]: etc

File: src/app/layout.tsx [hunk 5]
```diff
@@ -16,7 +18,11 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        {" "}
```
Comment (blbergo) [inline src/app/layout.tsx:22]: what is this space for?

File: src/app/page.tsx [hunk 6]
```diff
@@ -1,113 +1,11 @@
...
-          <p className="m-0 max-w-[30ch] text-balance text-sm opacity-50">
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
+export default function Page() {
+  return (
+    <div>
+      <HomePage />
```
Comment (blbergo) [inline src/app/page.tsx:8]: you can just move the contents from the homepage component into here, 1 layer to deep of abstraction imo

**Review by blbergo — APPROVED** [review]
overall, looks good. definitely improved since our last project :) 
address my comments and fix the merge conf and you are good to go!

File: src/app/page.tsx [hunk 7]
```diff
@@ -1,11 +1,13 @@
...
-import HomePage from "./homepage/page";
-
-export default function Page() {
+export default function HomePage() {
   return (
-    <div>
-      <HomePage />
-    </div>
+    <>
+      <Header />
```
Comment (mgkram4) [inline src/app/page.tsx:8]: is this how to do it ?

**Comment by mgkram4** [comment]
I made all the changes , but I cant seem to implement the homepage into the src/page without a merge conflict

**Pull request merged**